assets/js/base.js

```diff
@@ -0,0 +1,11 @@
+function handleAnchorClick(anchor) {
+   // write the full anchor URL to the clipboard
+   navigator.clipboard.writeText(location.href.split('#')[0] + '#' + anchor ); 
+
+   const checkmark = document.getElementById('checkmark-' + anchor);
+   checkmark.classList.add('visible');
+   
+   setTimeout(() => {
+     checkmark.classList.remove('visible');
```

Comment:
Can it say copied? A bit like the code samples do? That would be more using existing patterns